index.html

```diff
@@ -188,8 +188,38 @@ <h3 id="h3">LET'S CONNECT</h3>
                     </ul>
                 </div>
             </section>
-
         </main>
+
+        <footer>
+            <section class="contact-form">
+                <h1 id="contact-me">Contact me </h1>
```

Comment:
- [ ] [REQUIRED] Please, make sure not to use more than 1️⃣ one `h1` tag in the whole document for semantic puposes.